python/cudf/cudf/utils/ioutils.py

```diff
@@ -491,7 +491,7 @@
 ----------
 fname : str
     File path or object where the ORC dataset will be stored.
-compression : {{ 'snappy', 'ZSTD', None }}, default 'snappy'
+compression : {{ 'snappy', 'ZSTD', 'lz4', None }}, default 'snappy'
```

Comment:
PyArrow appears to support a different set of values (the casing is different for the values in common, the ordering is different, and `'UNCOMPRESSED'` is different than `None`):

> Valid values: {‘UNCOMPRESSED’, ‘SNAPPY’, ‘ZLIB’, ‘LZ4’, ‘ZSTD’}

https://arrow.apache.org/docs/python/generated/pyarrow.orc.ORCWriter.html
https://arrow.apache.org/docs/python/generated/pyarrow.orc.write_table.html

Reply:
I added ZLIB, which was missing from the list. 
Could you please explain why the order is relevant here?

IMO changes to other compression options (outside of docs) are outside of the scope of this PR. I feel like I already messed with them too much with `upper()`.
Should I open an issue for this? I think the changes are:

- Add `UNCOMPRESSED`;
- Change all strings to uppercase (this changes the default, which is "snappy" ATM).